packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-embedded-widget/main/react/basic-sign-in.md

```diff
@@ -0,0 +1,201 @@
+### Create routes
+
+Typically, an app contain routes that require authentication in order to render. Use the `<SecureRoute>` component from [Okta React SDK](https://github.com/okta/okta-react) to define authenticated routes for your app. The following are some basic routes that you need to configure for your app:
+
+* A [default page](#default-page-route) to handle basic control of the app
```

Comment:
I don't know how important the order of the routes in this list is, but it doesn't match up with the order that the topics that follow are presented.

This list:
- default page route
- login route
- callback route
- protected route

The order of topics:
- default page route
- protected route
- login route
- callback route

As someone that is new to the subject, it's mildly disorienting.

Reply:
Thanks for this - I'll move the sections around so that it matches :-)